server/src/main/java/io/seata/server/store/file/FileTransactionStoreManager.java

```diff
@@ -215,46 +215,87 @@ private boolean saveHistory() throws IOException {
             writeDataFileRunnable.putRequest(new CloseFileRequest(currFileChannel, currRaf));
             Files.move(currDataFile.toPath(), new File(hisFullFileName).toPath(), StandardCopyOption.REPLACE_EXISTING);
         } catch (IOException exx) {
-            LOGGER.error("save history data file error," + exx.getMessage());
+            LOGGER.error("save history data file error, {}", exx.getMessage(), exx);
             result = false;
         } finally {
             initFile(currFullFileName);
         }
         return result;
     }
 
+    private boolean writeDataFrame(byte[] data, int dataLength) {
```

Comment:
The dataLength passed in by the caller is data.length, whether to remove the dataLength parameter，use data.length??

Reply:
Function writeDataFrame is independent, has more functions than just specifying data as parameter, that could be more useful in future. I think no harm done to provide one more parameter dataLength. Do you agree?

Reply:
I am no problem, It’s just that developer are looking a little doubt by now.

Reply:
Ok, it's fine to remove dataLength parameter by now, clearer and more concise at least.

Reply:
Committed.